doc_src/cmds/switch.rst

```diff
@@ -19,6 +19,8 @@ Each ``case`` command is given one or more parameters. The first ``case`` comman
 
 Note that fish does not fall through on case statements. Only the first matching case is executed.
 
+Note that :ref:`break <cmd-break>` cannot be used to exit a case statement early like in other languages. It can only be used in loops.
```

Comment:
Can you change "case statement" to "case/switch block" to make it more searchable and easier for beginners to understand?

Reply:
Sure, done.